test/python/apps/ethereum.py

```diff
@@ -1,6 +1,12 @@
 from ragger.utils import pack_APDU, RAPDU
 from ragger.error import ExceptionRAPDU
 
+from ..utils import create_currency_config, create_sub_currency_config, pack_derivation_path
+
+ETH_CONF = create_currency_config("ETH", "Ethereum", create_sub_currency_config("ETH", 18))
```

Comment:
This is so much nicer than previous magic byte blobs!

Reply:
A step in the good direction !